Cakefile

```diff
@@ -29,6 +29,15 @@ header = """
 # Used in folder names like `docs/v1`.
 majorVersion = parseInt CoffeeScript.VERSION.split('.')[0], 10
 
+# Patterns of names of test files which depend on currently absent features
+testFilesToSkip = [
+  /async/           unless try new Function 'async ()   => {}'
```

Comment:
Shouldn’t this be `/^async$/` so that it doesn’t automatically include `async_iterators` as well?

Reply:
My reasoning was that any file with 'async' in its name probably requires the async feature, but I don't mind being more specific if you prefer.

Reply:
I think we should map feature detections to specific files, rather than relying on substrings in filenames. At quick glance it’s not obvious that `/async/` will match both `async.coffee` and `async_iterators.coffee`, and so this is a potential bug in the future when someone adds another one that overlaps with filenames they didn’t intend. I think all you need to do is change these regexes into strings? And then change `fileName.match` to be an equality check.